### ISSUE TITLE
batch size

### ISSUE THREAD

**Issue by frelenium** [vislearn/LessMore#9]
I was reading the paper and saw DSAC++ use a batch size of 1 but DSAC use 64. Why batch size 1 is necessary?

**Comment by ebrach** [comment]
Hi,
the DSAC network was processing patches independently, i.e. a batch size of 64 means 64 image patches. The DSAC++ network is a FCN which processes an image as a whole, i.e. a batch size of 1 means 1 whole image consisting of 60x80 (correlated) image patches. Due to memory constraints we where not able so increase the batch size for DSAC++ significantly, for example to train with batch norm. 
Kind regards,
Eric
